crates/hotshot/src/lib.rs

```diff
@@ -146,7 +146,7 @@ pub struct SystemContext<TYPES: NodeType, I: NodeImplementation<TYPES>> {
 
     /// Access to the output event stream.
     #[deprecated(
-        note = "please use the `get_event_stream` method on `SystemContextHandle` instead. This field will be made private in a future release of HotShot"
+        note = "please use the `event_stream` method on `SystemContextHandle` instead. This field will be made private in a future release of HotShot"
     )]
     pub output_event_stream: (Sender<Event<TYPES>>, InactiveReceiver<Event<TYPES>>),
```

Comment:
I think this can probably be removed now

Reply:
Just to double-check my understanding. Did you mean that we can remove `pub` and delete this note, or that we can actually remove this field?

Reply:
Updated assuming it's the former case, since it looks like we can't just remove this function.

Reply:
Oh yes, sorry! I think I had in my head "just delete it", but you're right -- it's a necessary field, so we just need to make it private.